docs/src/operate/troubleshooting/index.md

```diff
@@ -6,6 +6,6 @@ sidebar_position: 8
 
 import DocCardList from '@theme/DocCardList';
 
-How to trouble shoot a thin-edge device
+How to trouble shoot a %%te%% device
```

Comment:
An explicit page description is required here (inappropriate wording, unexpected %%te%% tag)

Reply:
Resolved by c8ea100c5cc37dd185f5cc3be9e144ce145874a7